guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -467,6 +467,13 @@ You can invoke `test_jdbcmysql`, `test_jdbcsqlite3` or `test_jdbcpostgresql` als
 
 The test suite runs with warnings enabled. Ideally, Ruby on Rails should issue no warnings, but there may be a few, as well as some from third-party libraries. Please ignore (or fix!) them, if any, and submit patches that do not issue new warnings.
 
+### Updating the documentation
+The Ruby on Rails [guides](https://guides.rubyonrails.org/) are designed to make developers immediately productive with Rails, and to help them understand how all of the pieces fit together.
+
+If your PR has added a new feature, or changed how an existing feature behaves, take a look at the guide for the impacted libraries and update the corresponding sections (or add a new one).
+
+For example, if your PR added [proxying to active storage](https://github.com/rails/rails/pull/34477), you would need to add a new section to the active storage guide explaining how to use that. 
```

Comment:
I'd avoid adding PRs as example to the guide. What makes that PR the best example? It is hard to tell. If we want to give examples, better to write it here.